docs/en/src/define_simple_object.md

```diff
@@ -8,6 +8,7 @@ The example below defines an object `MyObject` which includes the fields `a` and
 use async_graphql::*;
 
 #[derive(SimpleObject)]
+#[graphql(desc = "An optional description")]
```

Comment:
I recommend this way more. 😁

```rust
/// An optional description
#[derive(SimpleObject)]
struct MyObject {
}
```

Reply:
Unfortunately it doesn't show up in the schema or the graphql playground

Reply:
> Unfortunately it doesn't show up in the schema or the graphql playground. This is handy for clients and consumers of the graphql service

In fact it can. 😁

Reply:
Ah I see, requires a bit of looking deeply. Can we add something to the docs to make this more clear?

Reply:
The `desc` attribute will soon be removed and will only support rustdoc in the future.😃

Reply:
Perhaps you could help us add a chapter on how to add a description?😊